components/transactions/TransactionItem.vue

```diff
@@ -133,12 +215,45 @@ export default {
       ].includes(this.transaction.type)
     },
     amounts() {
-      if (this.transaction.details.amounts) {
+      if (
+        this.transaction.details.amounts &&
+        this.transaction.type !== lunieMessageTypes.SEND_MULTIPLE) { // eslint-disable-line
         return this.transaction.details.amounts
       } else if (this.transaction.details.amount) {
         return Array.isArray(this.transaction.details.amount)
           ? this.transaction.details.amount
           : [this.transaction.details.amount]
+
+        // sendMultiple:
+      } else if (
+        this.transaction.rawMessage.message.inputs &&
+        this.transaction.rawMessage.message.inputs[0].address === this.session.address // eslint-disable-line
+      ) {
+        const totalAmount = this.transaction.rawMessage.message.inputs[0].coins; // eslint-disable-line
+        const amountInLIKE = []
+        const sendAmount = {}
+        if (totalAmount) {
+          sendAmount.amount = totalAmount[0].amount / 1e9
+          sendAmount.denom = 'LIKE'
```

Comment:
Should avoid hardcode and just get the `denom` from payload (`totalAmount[0].denom`?)

Reply:
because the denom is 'nanolike', but here I would want 'LIKE'

Reply:
redo the logic at cosmos-reducer